client/src/composables/fileDrop.ts

```diff
@@ -41,6 +47,7 @@ export function useFileDrop(
                 case "dragstart":
                     return "blocked";
                 case "dragenter":
+                    setIframeEvents(iframesNoInteract, true);
                     if (!(unref(solo) && isAnyModalOpen())) {
                         return "fileDragging";
```

Comment:
```suggestion
                    if (!(unref(solo) && isAnyModalOpen())) {
                        setIframeEvents(iframesNoInteract, true);
                        return "fileDragging";
```

Having the `setIframeEvents` function outside of the if block could break iframes, as it may disable pointer events without the state changing to a dragging state.

Reply:
I had wondered something similar - so I may need to adjust the state machine so that it provides more information to the client component so that I can move the call to `setIframeEvents` to the client component instead.

Reply:
@ElectronicBlueberry Updated with `watch` method in [this commit](https://github.com/galaxyproject/galaxy/pull/16995/commits/45b32f8cdaf7c11f92e344a122d716df3d796d78).

Reply:
Thank you, that is a very clean solution!